tests/semanticdb/todo

```diff
@@ -0,0 +1,13 @@
+- [ ] Hide unapply, unapplySeq?
+- [ ] Hide constructor of template
+- [ ] Hide local vars in pattern def?
+- [ ] Hide desugaring of for comprehension
+- [ ] Hide calls to implicit conversions and their arguments
+- [ ] Hide Tuple contructors
+- [ ] Hide qualifier in Select inside a type?
+- [x] Hide `Predef.classOf`
+- [x] Hide `scala.Predef`
+- [x] Hide `scala.StringContext`
```

Comment:
It's fine to include `StringContext` references as long as they have zero length ranges. In Scala 2 we emit `Synthetic` https://scalameta.org/docs/semanticdb/specification.html#synthetic for inferred implicits, `.apply`, expanded for comprehensions, etc.

Reply:
Thanks for the tip, I got the same effect by just checking the range length of the prefix.